core/migrations/0005_geo_api_keys.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.10 on 2018-06-12 17:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0004_agreement_model'),
```

Comment:
Travis plendas ke la dependeco ne estas `0005_geo_api_keys` kaj ke la numero de tiu migrado ne estas `0006`.

Verŝajne en via komputilo faru tiel:
1. Rebazigu el `master` se ne jam farite
1. Portempe forprenu tiun dosieron
1. Malfaru la migradon ĝis `5`: `./manage.py migrate core 0005_agreement_withdrawal`

1. Remetu tion dosieron kun ĝusta numero kaj dependeco.
1. `./manage.py migrate core`

Se ne funkcias, provu malfari la migradon ĝis `4`: `./manage.py migrate core 0004_agreement_model`  kaj reprovu.